### PR TITLE
PAR-2683: Display Product Protection Offer on PDP

### PR DESCRIPTION
**Pull request by scott-taylor-extend**
# Description

This is the product protection offer for simple products, configurable products and grouped products

## Ticket

[PAR-2683](https://helloextend.atlassian.net/browse/PAR-2683)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactor (new code changes but everything functions the same)

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have deployed and validated my code on a sandbox
- [ ] I have added tests that prove my fix is effective or that my feature works

[PAR-2683]: https://helloextend.atlassian.net/browse/PAR-2683?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by scott-taylor-extend** [comment]
It seems merging in master is being treated a new files and I'm not sure why, but the only files that need to be reviewed are the files in view/frontend, excluding shipping-protection-offer.js, which can be ignored.

File: view/frontend/layout/checkout_index_index.xml [hunk 1]
```diff
@@ -1,17 +1,10 @@
...
   -->
 <page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="1column" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd">
     <body>
-        <referenceContainer name="content">
-            <block name="extend-environment-store-uuid" template="Extend_Integration::checkout/sp-quote-config.phtml">
-                <arguments>
-                    <argument name="viewModel" xsi:type="object">Extend\Integration\ViewModel\EnvironmentAndExtendStoreUuid</argument>
-                </arguments>
-            </block>
-        </referenceContainer>
```
Comment (jay-faust-extend) [inline view/frontend/layout/checkout_index_index.xml]: Why was this removed?

**Review by jay-faust-extend — DISMISSED** [review]

**Review by milesmcleod — DISMISSED** [review]

**Review by jay-faust-extend — APPROVED** [review]

**Review by milesmcleod — APPROVED** [review]
